.circleci/config.yml

```diff
@@ -15,27 +15,13 @@ jobs:
       - run:
           name: Check that scipy links to OpenBLAS
           command: docker run ds-python python -c "from scipy.linalg import _fblas"
-      - run:
-          name: Verify that numpy and matplotlib function
-          command: docker run ds-python python -c "import numpy, os; import matplotlib.pyplot as plt; x = numpy.arange(100); y = numpy.sin(x); plt.plot(x, y);"
-      - run:
-          name: Check that seaborn is installed
-          command: docker run ds-python python -c "import seaborn"
       - run:
           name: Check that numpy imports from bash shell
           command: docker run -t ds-python /bin/bash -c "python -c 'import numpy'"
-      - run:
-          name: Check that tensorflow imports from bash shell
-          command: docker run ds-python /bin/bash -c "python -c 'import tensorflow'"
       - run:
           name: Verify that numpy does not link to MKL
           command: docker run ds-python python -c "from numpy.distutils import system_info; assert system_info.get_info('mkl') == {}"
+
       - run:
           name: Run numpy unit tests
-          command: docker run ds-python python -c "import numpy; numpy.test()"
-      - run:
-          name: List packages installed from conda-forge
-          command: docker run ds-python /bin/bash -c "conda list|grep conda-forge"
-      - run:
-          name: Check for conda-pip conflicts
-          command: docker run ds-python /bin/bash -c "conda list|python -c \"import sys; from collections import Counter; duplicates = {k for k, v in Counter([l.split()[0] for l in sys.stdin.readlines() if not l.startswith('#')]).items() if v>1}; assert not duplicates, f'These packages are installed by both pip and conda:\n{duplicates}'\""
+          command: docker run ds-python /bin/bash -c 'pip install pytest hypothesis; python -c "import numpy; numpy.test()"'
```

Comment:
Maybe we should add a test to make sure importing civis works? It could be something as simple as `docker run ds-python /bin/bash -c "python -c 'import civis'"`? Just in case somehow we didn't catch some weird/incompatible interactions between civis-python and any other installed packages in this image...